web_src/less/_base.less

```diff
@@ -1182,3 +1182,7 @@ i.icon.centerlock {
 .ui.popup .ui.label {
     margin-bottom: 0.4em;
 }
+
+.invisible {
```

Comment:
Maybe .hidden instead of .invisible ? Doesn't semantic has a class already for this purpose ?

Reply:
👍 to hidden, but I would not introduce a dependency on semantic here, even if it has such a class.

Reply:
Semantic's `hidden` uses `display: none;` which causes some jumping, that's the only reason I opted for this instead.  
I can change it to `.hidden` though I'm unsure if that will cause collisions in some pages. Probably not since I doubt semantic does much with SVGs and afaict it doesn't have a generic `.hidden {}` selector

Reply:
I guess keeping `.invisible` is fine in that case. `display:none` is bad for a11y also, so should not be used.

Reply:
Yes let this at invisible so.